enigma-js/test/Enigma.spec.js

```diff
@@ -2466,7 +2466,18 @@ describe('Enigma tests', () => {
       const report = '0x' + Array.from(workerData[1]).map((c) => c.charCodeAt(0).toString(16)).join('');
       const signature = '0x' + workerData[3];
       const proof = utils.hash([enigmaUpgradedContract.options.address]);
-      const sig = await enigma.web3.eth.sign(proof, worker.account);
+      function fixSignature (signature) {
```

Comment:
Can you further comment on this in the description of the PR, there is no mention of it there.